src/domUpdates.js

```diff
@@ -128,36 +128,59 @@ const generateRandomUser = users => {
 }
 
 function showSearchResults() {
-  let searchValue = searchInput.value
-  removeHiddenClass([allRecipeDisplay, allFilterDisplay])
+  let searchValue = searchInput.value;
+  removeHiddenClass([allRecipeDisplay, allFilterDisplay]);
   addHiddenClass([frontRecipeDisplay]);
-    allRecipeDisplay.innerHTML = ''
-    getData('recipes').then(({recipes}) => {
-    recipesfromName(recipes, searchValue).forEach(recipe => allRecipeDisplay.innerHTML += `<div class = "recipe-wrapper">
-    <img id="${recipe.name}" src="${recipe.image}" class="recipe">
-    <div class = "recipe-info">
-      <p>${recipe.name}</p>
-    </div>`)
-    })
+  allRecipeDisplay.innerHTML = '';
+  getData('recipes').then(({ recipes }) => {
+    const searchedRecipes = recipesfromName(recipes, searchValue);
+    if (searchedRecipes.length === 0) {
+      allRecipeDisplay.innerHTML = `
+        <div class="no-recipe-found-message">
+          <p>Sorry, ${currentUser.name}, we currently don't have any matching recipes.</p>
+        </div>`;
```

Comment:
Nice job adding a conditional for this function! Makes the website more user friendly